workflow-templates/im-test-sam-yaml.yml

```
@@ -0,0 +1,52 @@
+# Workflow Code: DigitalWerewolf_v1    DO NOT REMOVE
+# Purpose:
+#    Runs the SAM YAML Validation action and then imports the
+#    output JSON into the Stewardship Mapping document repository
+#
+# Frequency:
+#    - This workflow should only be used once per repository
+#
+# Projects to use this Template with:
+#    -  Any project that needs to report SAM information for Stewardship Mapping
+#       See https://github.com/im-practices/master-architecture-plan/blob/main/Implementation%20Decisions/adr-015_system_architecture_metadata.md
+#       for more information
+#
+# TODO: Prerequisites:
+#    - Create a sam.yaml file that describes the Bounded Context projet information
+
+name: SAM.yaml Test and Import to Stewardship Mapping
+
+on:
+  # After a PR is merged, the workflow will check for changes to the SAM YAML file in the root directory
+  pull_request:
+    types: [closed]
+    paths:
+      - 'sam.yaml' # TODO - Verify sam.yaml file name and path. This path is for the a sam.yaml file located at the root of the proejct
```

This has a typo in project...but it might be nice to clarify that you can't use the relative './' here or it won't trigger properly.
```suggestion
      - 'sam.yaml' # TODO - Verify sam.yaml file name, extension and path. This path is for the a sam.yaml file located at the root of the project
```

---

Maybe...
> \# TODO - Verify sam.yaml file name, extension and path. This path is for ~the~a sam.yaml file located at the root of the project.<ins> Note it does not start with '/' nor './'.</ins>

---

Just wondering what would happen if it were
```YAML
    paths:
      - '**/sam.yaml'
```

---

I tested this.  It will pick up a sam.yaml anywhere in the project.  This could introduce problems if people include sam.yamls in a lot of different places.